docs/user-guide.md

```diff
@@ -96,6 +96,7 @@ The `@NullMarked` annotation indicates that references can't be null in its scop
 unless their types are explicitly marked `@Nullable`. If applied to a package then
 its scope is all the code in the package. If applied to a class or interface
 then its scope is all the code in that class or interface.
+TODO: mention modules. First use of `interface`, before `class` subsumed all type declarations.
```

Comment:
Maybe just say package/module to also cover modules.
The extra mention of `interface` here is a bit surprising, as before (and according to our glossary), `class` covers all type declarations.